pkgs/development/libraries/libcec/default.nix

```diff
@@ -14,7 +14,11 @@ stdenv.mkDerivation {
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ cmake udev libcec_platform ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=1" ];
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=1" ] ++
+    stdenv.lib.optionals (libraspberrypi != null) [
+      "-DRPI_INCLUDE_DIR=${libraspberrypi}/include"
```

Comment:
Usually cmake can use pkg-config to find these directories. Are you sure it's necessary here?

Reply:
Thanks for point this out! For some reason I'd assumed that the raspberry pi libraries didn't follow that standards well, but it turns out they do just fine.